python/GafferUI/Layouts.py

```diff
@@ -132,7 +132,7 @@ def save( self, fileObject, nameRegex = None ) :
 
 		# finally write out the layouts
 		for name in namesToWrite :
-			fileObject.write( "layouts.add( \"%s\", \"%s\" )\n\n" % ( name, self.__namedLayouts[name] ) )
+			fileObject.write( "layouts.add( \"%s\", \"%s\" )\n\n" % ( name, self.__namedLayouts[name].replace( '"', '\\"' ) ) )
```

Comment:
Rather than escape manually, I wonder if this would be better?

```
fileObject.write( "layouts.add( {0}, {1} )\n\n".format( ( repr( name ), repr( self.__namedLayouts[name] ) ) )
```

That's effectively how we serialise values for StringPlugs, and I think it should correctly escape everything, not just double quotes...